en/docs/apis/application-rest-api.md

```diff
@@ -3,13 +3,13 @@ template: templates/swagger.html
 ---
 # Application Management Rest API Definition - v1
 
-!!! Note
+??? Note "Click for instructions"
```

Comment:
in the new structure, we want to avoid expandable blocks as much as possible right?

Reply:
AFAIR, in the last discussion we decided that we will be using both expandable blocks as well as tabbed code blocks since we don't use the pdf feature anymore. We agreed to use these for repetitive and optional content.